crates/rustc_codegen_spirv/Cargo.toml

```diff
@@ -56,7 +56,7 @@ sanitize-filename = "0.4"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 smallvec = { version = "1.6.1", features = ["union"] }
-spirv-tools = { version = "0.9", default-features = false }
+spirv-tools = { version = "0.10", default-features = false }
```

Comment:
If you move this to `Cargo.toml` and have `spirv-tools.workspace = true` instead here, it would theoretically be possible to have the CI script not hardcode `0.10.0`, and instead scan `Cargo.toml` for the version.

There's also stuff like this (`jq` would ofc be replaced by e.g. using the `cargo_metadata` crate etc.):
```console
$ cargo tree -p spirv-tools --depth 0
spirv-tools v0.10.0
$ cargo metadata --format-version 1 | jq -r '.packages[] | select(.name == "spirv-tools").version'
0.10.0
```

For now it seems fine to land this PR as-is, just thinking out loud mostly.